src/validations.ts

```diff
@@ -0,0 +1,92 @@
+import validator from "validator";
+import {
+	BankAccount,
+	DataModel,
+	SimplePayment,
+} from "./types.js";
+
+export enum ValidationErrorMessage {
+	InvalidIBAN = "Invalid IBAN",
+	InvalidBIC = "Invalid BIC",
+	InvalidCurrencyCode = "Invalid currency code",
+	InvalidDate = "Invalid date",
```

Comment:
You can remove `Invalid` prefix, it's redundant

Reply:
Try to specify not just which field is invalid, but also what type of input would be valid. For example: 

"Invalid IBAN. Make sure you use `ISO 13616` format with the pattern `[A-Z]{2}[0-9]{2}[A-Z0-9]{0,30}`"

Reply:
[7be3324](https://github.com/xseman/bysquare/pull/43/commits/7be3324ded27e3b5e0a6de07e1fc4528c4e3a7ce) . I made the message a bit shorter. I think fields like amount without a regex would be too verbose.